contracts/modules/capital/SwapOperator.sol

```diff
@@ -48,16 +55,28 @@ contract SwapOperator {
   uint public constant MIN_TIME_BETWEEN_ORDERS = 900; // 15 minutes
   uint public constant maxFee = 0.3 ether;
 
+  // Safe variables
+  address public safe;
+  Request public transferRequest;
+  mapping(address => bool) public allowedAssets;
```

Comment:
needs a comment explaining where are these assets allowed, allowed to what?

Reply:
Or maybe even rename to include "investment". `allowedInvestmentAssets`? Don't necessarily stick to this suggestion, discuss please.

Reply:
Maybe something more safe related, since it is what we are sending to the safe (`allowedSafeTransferAssets`), if we in future have different investment flows we will need to change it again.

Reply:
I'd go for `allowedAssetsForSafeTransfer` as it's more explicit, but up to you